presto-spark-base/src/test/java/com/facebook/presto/spark/execution/http/TestPrestoSparkHttpClient.java

```diff
@@ -1027,10 +957,8 @@ else if (method.equalsIgnoreCase("DELETE")) {
                                     future.completeExceptionally(e);
```

Comment:
While you are at it, do you mind DRY'ing out this block as well into a function
```Java
try {
   future.complete(responseHandler.handle(
        request,
            responseManager.createResultResponse()));
}
catch (Exception e) {
      e.printStackTrace();
      future.completeExceptionally(e);
}
```

Reply:
Refactored by moving try-catch block outside of the if-else branches.